templates/api.html

```diff
@@ -1,43 +1,137 @@
-Mysterium.Network API<br/>
-Version: 1.0<br/>
-URL Base: https://mvp.mysterium.network:5000<br/>
-Params should be encoded in JSON and passed as HTTP body data.<br/>
-Returned values are encoded in JSON.
-API reqest returns status code. For example status code 200 means success.<br/>
-
-<p>
-    POST /v1/node_register<br/>
-    Params: Proposal<br/>
-    Proposal Example:
-    "service_proposal": {
-            "id": 1,
-            "format": "service-proposal/v1",
-            "provider_id": "node1"
+<!DOCTYPE html>
+<html lang="en-US">
+<head>
+    <meta charset="utf-8">
+    <title>Endpoints</title>
+    <style>
+        .endpoint {
+            background-color: #e8e8e8;
+            padding: 15px;
+            margin-top: 20px;
+        }
+        .endpoint .title {
+            font-weight: bold;
+        }
+    </style>
+</head>
+
+<body>
+
+<div>Most parameters should be encoded in JSON and passed as HTTP body data.</div>
+<div>Returned values are encoded in JSON.</div>
+<div>Each endpoint returns status code. For example status code 200 means success.</div>
+<div>For POST methods Authorization header with signature is required.</div>
+
+<div class="endpoint">
+    <div class="title">POST /v1/register_proposal</div>
+    <div>Registers proposal to discovery service.</div>
+    <div>Example JSON Parameters:</div>
+    <pre>
+    {
+        "service_proposal": {
+                "format": "service-proposal/v1",
+                "service_type": "openvpn",
+                "provider_id": "0x0000000000000000000000000000000000000001"
+        }
+    }
+    </pre>
+    <div>Returns:</div>
+    <pre>
+    {}
+    </pre>
+</div>
+
+<div class="endpoint">
+    <div class="title">POST /v1/unregister_proposal</div>
+    <div>Unregisters proposal from discovery service.</div>
+    <div>Example JSON Parameters:</div>
+    <pre>
+    {
+        "service_type": "openvpn",
+        "provider_id": "0x0000000000000000000000000000000000000001"
+    }
+    </pre>
+    <div>Returns:</div>
+    <pre>
+    {}
+    </pre>
+</div>
+
+<div class="endpoint">
+    <div class="title">GET /v1/proposals[?node_key=...&service_type=...]</div>
+    <div>Returns list of proposals.</div>
+    <div>URL parameters:</div>
+    <div>node_key (optional), example: "0x0000000000000000000000000000000000000001"</div>
+    <div>service_type (optional), example: "openvpn"</div>
+    <div>Return example:</div>
+    <pre>
+    {
+        "proposals":[
```

Comment:
this example is not really representative of the response

if you look at https://testnet-api.mysterium.network/v1/proposals we do have quite a bit more of information there than the small object you represent here

Reply:
example updated